docs/content/latest/api/ycql/ddl_create_table.md

```diff
@@ -96,15 +96,24 @@ Where
 - Columns in the primary key cannot be static.
 - A table without clustering columns cannot have static columns (without clustering columns the primary key and the partition key are identical so static columns would be the same as regular columns).
 
-### TABLE PROPERTIES
+### *table_properties*
 
 - The `CLUSTERING ORDER BY` property can be used to set the ordering for each clustering column individually (default is `ASC`).
 - The `default_time_to_live` property sets the default expiration time (TTL) in seconds for a table. The expiration time can be overridden by setting TTL for individual rows. The default value is `0` and means rows do not expire.
 - The `transactions` property specifies if distributed transactions are enabled in the table. To enable distributed transactions, use `transactions = { 'enabled' : true }`.
-- The `tablets` property specifies the number of tablets to be used. This is useful for two data center (2DC) deployments. See example below: [Create CDC table specifying number of tablets](#create-cdc-table-specifying-number-of-tablets)
 - Use the `AND` operator to use multiple table properties.
 - The other CQL table properties are allowed in the syntax but are currently ignored internally (have no effect).
 
+### *tablets_properties*
```

Comment:
There is no tablet property.

Specifying number of tablets for a table is still a table property and should be included in the section above.

Reply:
I forgot to ask you about that -- it looked like the diagram was wrong. I had added it to the section above before, but thought that was wrong.  Adding that section back.